stubs/inertia/resources/js/Jetstream/Modal.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <portal to="modal">
+    <Teleport to="#modal">
```

Comment:
I haven't tested this, but do you really need the `<div id="modal"></div>`? I think you can just do `to="body"`. Also, I'm pretty sure even Vue uses the lowercase version of `<teleport>` in their docs, so that might be more idiomatic.
```suggestion
    <teleport to="body">
```